src/main/java/SubstitutionTool.java

```diff
@@ -1,14 +1,130 @@
 import java.io.InputStream;
+import java.util.Scanner;
 
 /**
  * Created by wessel on 9/9/16.
  */
 class SubstitutionTool {
-    static StringBuilder decrypt(InputStream in, String key) {
-        throw new UnsupportedOperationException("Not jet implemented!");
+
+    private final static int START = (int) 'a';
+    private final static int END = (int) 'z';
+    private final static int CAPTITAL_START = (int) 'A';
+    private final static int CAPTITAL_END = (int) 'Z';
+    private boolean decryptMode;
+    private boolean oMode;
+
+    public void crypter(InputStream in, String key, boolean oMode, boolean decryptMode){
+        this.oMode = oMode;
+        this.decryptMode = decryptMode;
+        Scanner s = null;
+        if (decryptMode) {
+            System.out.println("decrypt: ");
+        } else {
+            System.out.println("encrypt: ");
+        }
+        String str;
+        try {
+            s = new Scanner(in);
+
+            while (s.hasNext()) {
+                if(!oMode){
+                    str = s.next().toLowerCase();
+                } else{
+                    str = s.next();
+                }
+                if (key.length() <= 2 && key.length() != 0) {
```

Comment:
De key length kan best langer zijn dan 2 characters, zolang het maar een integer is.